ckanapi/cli/main.py

```diff
@@ -4,6 +4,7 @@
   ckanapi action ACTION_NAME
           [(KEY=STRING | KEY:JSON | KEY@FILE ) ... | -i | -I JSON_INPUT]
           [-j | -J] [[-c CONFIG] [-u USER] | -r SITE_URL [-a APIKEY] [-g]]
+          [--no-verify]
```

Comment:
Let's follow the `curl` convention and use `--insecure` instead

Reply:
ah, good call... I'll do that